services/director/temp_generate_openapi.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -e
+
+cd $(dirname $0)
+usage()
+{
+    echo "usage: temp_generate_openapi.sh [[[-i input]] | [-h help]]"
+}
+
+apihub_specs_dir=
+# process arguments
+while [ "$1" != "" ]; do
+    case $1 in
+        -i | --input )          shift
+                                apihub_specs_dir=$1
+                                ;;
+        -h | --help )           usage
+                                exit
+                                ;;
+        * )                     usage
+                                exit 1
+    esac
+    shift
+done
+
+if [ -z "$apihub_specs_dir" ]; then
+    echo "please define an apihub specs directory..."
+    usage
+    exit 1
+fi
+
+docker run \
+    -v $apihub_specs_dir:/input \
+    -v ${PWD}/src/simcore_service_director/oas3/v0:/output \
+    itisfoundation/oas_resolver \
```

Comment:
all tooling containers should be defined in https://github.com/ITISFoundation/dockerfiles 
should move there also itisfoundation/oas_resolver be

Reply:
yep, but this will disappear once the director is re-made using the cookiecutter